packages/flutter_tools/lib/src/isolated/devfs_web.dart

```diff
@@ -181,6 +182,8 @@ class WebAssetServer implements AssetReader {
     required WebRendererMode webRenderer,
     bool testMode = false,
     DwdsLauncher dwdsLauncher = Dwds.start,
+// TODO(markzipan): Make sure this default value aligns with that in the debugger options.
+    bool ddcModuleSystem = false,
```

Comment:
The alignment of this TODO comment, and the several appearances of `ddcModuleSystem` below looks wacky, is it github messing up or does this need a few spaces?

Reply:
That's very odd. Something about the diffing tool in VSCode must have wonked out. Updated.